src/sentry/workflow_engine/models/alertruletrigger_data_condition.py

```diff
@@ -0,0 +1,14 @@
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import DefaultFieldsModel, FlexibleForeignKey, region_silo_model
+
+
+@region_silo_model
+class AlertRuleTriggerDataCondition(DefaultFieldsModel):
+    """
+    A lookup model for alertruletriggers and dataconditions.
+    """
+
+    __relocation_scope__ = RelocationScope.Excluded
+
+    alert_rule_trigger = FlexibleForeignKey("sentry.AlertRuleTrigger")
+    data_condition = FlexibleForeignKey("workflow_engine.DataCondition")
```

Comment:
🤔 should this be to DataConditionGroup? (i can't remember if there are many statements on an alert_rule_trigger?)